src/components/tests/__snapshots__/QuickOpenModal.spec.js.snap

```diff
@@ -343,7 +343,7 @@ exports[`QuickOpenModal Basic render with mount 1`] = `
               placeholder="Search sources…"
               query=""
               selectedItemId=""
-              showErrorEmoji={true}
+              showErrorEmoji={false}
```

Comment:
The snapshots might need a rework after the logic is done since we are shifting the checks to a different place and `showErrorEmoji` will have different values.